ethereum/core/src/test/java/tech/pegasys/pantheon/ethereum/worldstate/DefaultMutableWorldStateTest.java

```diff
@@ -482,8 +481,8 @@ public void replaceAccountCode() {
     final WorldUpdater updater = worldState.updater();
     final MutableAccount account = updater.createAccount(ADDRESS);
     account.setBalance(Wei.of(100000));
-    account.setCode(BytesValue.of(1, 2, 3), Account.DEFAULT_VERSION);
-    account.setCode(BytesValue.of(3, 2, 1), Account.DEFAULT_VERSION);
+    account.setCode(BytesValue.of(1, 2, 3));
```

Comment:
(Discussion) The version is not set in the test, should we set explicitly the version and check it ?

Reply:
done